src/miner.h

```diff
@@ -24,7 +24,8 @@ class CWallet;
 namespace Consensus { struct Params; };
 
 static const bool DEFAULT_GENERATE = false;
-static const int DEFAULT_GENERATE_THREADS = 1;
+static const int DEFAULT_GENERATE_THREADS = -1;
```

Comment:
Rename to DEFAULT_GENERATE_THREADS_CPU to make it real obvious :D

Files where it is used are init.cpp/rpcmining.cpp/miner.h